### ISSUE TITLE
Code Documentation

### ISSUE THREAD

**Issue by jryates** [wannier-developers/wannier90#4]
From the discussion a proposal is to keep the user guide up to date - as a list of input parameters, functionality, specification of input and output files.

It was suggested to use FORD to document modules and subroutines (useful for coders)
https://pypi.python.org/pypi/FORD

**Comment by jryates** [comment]
First draft of the documentation is done. All modules have a description. So do all subroutines with the exception of sitesymmetry and transport. Most of the module variables and subroutine inputs are done - but there are some notable exceptions (e.g. parameters). So at the moment I think it is useful enough to be advertised, but more work is needed.

**Comment by giovannipizzi** [comment]
I agree. Should we remove the section with the graph with the relations between the various modules in the docs (that is probably also a bit outdated) and just point to the FORD docs?

**Comment by jryates** [comment]
I agree that we remove it.
However, unfortunately the module usage graphs are wrong in FORD - as it only reports modules that are globally used in a module (so those used in a subroutine within a module are missed). The call graphs within a subroutine are correct.

**Comment by mostofi** [comment]
+ Prioritise parameters for v2.1
+ Then if time move on to site symmetry and transport

**Comment by giovannipizzi** [comment]
For 4.0.3 we will just check that it's linked appropriately and easy to find

**Comment by jryates** [comment]
https://wannier.org/ford/index.html 
Last updated 2020.

**Comment by giovannipizzi** [comment]
#697 reinstates FORD and puts in the CI (with docs on GitHub pages).
When #697 is merged we close this - a separate PR can extend the documentation to missing code
